include/unifex/any_unique.hpp

```diff
@@ -409,7 +169,10 @@ class _byval<CPOs...>::type
   }
 
  private:
-  using vtable_holder_t = vtable_holder<_deallocate_cpo, CPOs...>;
+  using vtable_holder_t = std::conditional_t<
+    (sizeof...(CPOs) <= 2),
```

Comment:
The idea here was to keep the `any_unique` size below 4x pointers so that it fits inside a cache-line (or half a cache-line).
We have 2 user-defined CPOs and the deallocate CPO in the vtable, plus the pointer to the object (= 4 pointers).

It is 3 for the any_ref because it doesn't need to store a deallocate entry in the vtable.